tests/testthat/test-ps_subsampling.R

```diff
@@ -39,7 +39,7 @@ test_that("Subsampling cohort throws no error", {
       maxCohortSizeForFitting = 10,
       control = cyclops_control
     ),
-    NA
+    NA # "NA" asserts that there should be no errors
```

Comment:
@azimov I am not sure if I understood your previous comment, but I think this additional comment would help?

Reply:
I follow,  this makes it clearer